### PR TITLE
Update search-engine-optimization-quiz.md

### PR DESCRIPTION
**Pull request by MichaelNaghavipour**
Answered all questions with no answers. Deleted duplicate questions.

### CODE REVIEW

File: seo/search-engine-optimization-quiz.md [hunk 1]
```diff
@@ -256,20 +256,20 @@
...
 - [ ] RSS
-- [ ] CTR
+- [x] CTR
 - [ ] PPC
 - [ ] LSI
 
 #### Q30. How do keywords change as people progress through the buying cycle?
 
-- [ ] They move from simple to more comples as searchers gather more information and then refine their choices.
+- [x] They move from simple to more complexas searchers gather more information and then refine their choices.
```
Comment (ruslanbes) [inline seo/search-engine-optimization-quiz.md:265]:
```suggestion
- [x] They move from simple to more complex as searchers gather more information and then refine their choices.
```

**Review by Ebazhanov — APPROVED** [review]
Warm welcome @MichaelNaghavipour! Thanks you for your improvements and answers 👍 🥇

**Comment by Ebazhanov** [comment]
@all-contributors please add @MichaelNaghavipour for design, code

**Comment by MichaelNaghavipour** [comment]
> Warm welcome @MichaelNaghavipour! Thanks you for your improvements and answers 👍 🥇

You're welcome @Ebazhanov! I do my best to help as much as I can.

**Pull request merged**